article.md

```diff
@@ -3,5 +3,24 @@ This is my first article
 
 Introduction
 
+Personal information
+
 My name is lili el khalil
-Master student at UdeM
+Born in 03 august 1998, my birthday is in 4 days !!
```

Comment:
haha! Good to know!! I am going to let people know on discord. But I suggest to remove your exact birthday from this document. It is silly, but birthday is a security question in many services, so you should not make this information too easy to find on the web.